Drop ED errno value, not actually useful, and poses FTBFS problems.

http://lists.gnu.org/archive/html/bug-hurd/2010-10/msg00053.html

---
 errno.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/sysdeps/mach/hurd/bits/errno.h.original	2010-11-01 16:18:08.000000000 +0100
+++ b/sysdeps/mach/hurd/bits/errno.h	2010-11-01 16:18:19.000000000 +0100
@@ -182,8 +182,10 @@
 #define	EBACKGROUND     _HURD_ERRNO (100)/* Inappropriate operation for background process */
 	EDIED           = _HURD_ERRNO (101),
 #define	EDIED           _HURD_ERRNO (101)/* Translator died */
+#if 0
 	ED              = _HURD_ERRNO (102),
 #define	ED              _HURD_ERRNO (102)/* ? */
+#endif
 	EGREGIOUS       = _HURD_ERRNO (103),
 #define	EGREGIOUS       _HURD_ERRNO (103)/* You really blew it this time */
 	EIEIO           = _HURD_ERRNO (104),
